test/functional/apps/vis_builder/_base.ts

```diff
@@ -42,19 +41,5 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       const isEmptyWorkspace = await PageObjects.visBuilder.isEmptyWorkspace();
       expect(isEmptyWorkspace).to.be(true);
     });
-
-    it('should show warning before changing visualization type', async () => {
-      await PageObjects.visBuilder.selectVisType('metric', false);
-      const confirmModalExists = await testSubjects.exists('confirmVisChangeModal');
-      expect(confirmModalExists).to.be(true);
-
-      await testSubjects.click('confirmModalCancelButton');
-    });
-
-    it('should change visualization type', async () => {
-      const pickerValue = await PageObjects.visBuilder.selectVisType('metric');
-
-      expect(pickerValue).to.eql('Metric');
-    });
```

Comment:
Deleted these tests because we now have cypress tests for these. Will remove all the others after 2.7